s4e-web/src/environments/environment.hmr.private.ts

```diff
@@ -5,10 +5,21 @@
 import sharedEnvironment from './base';
 
 export const environment = {
+  //// TODO: Propagate those changes on every environment in base.ts file
+  backendDateFormat: 'YYYY-MM-DDTHH:mm:ss[Z]',
+  generalErrorKey: '__general__',
+  apiPrefixV1: 'api/v1',
+  projection: {toProjection: 'EPSG:3857', coordinates: [19, 52] as [number, number]},
+  maxZoom: 12,
+  timezone: Intl.DateTimeFormat().resolvedOptions().timeZone,
+  ////
+
```

Comment:
I think it's a good moment to move this common environment variables to separate file eg. `environment.common.ts`:

```
# environment.common.ts

export const commonEnvironment = {
    backendDateFormat: 'YYYY-MM-DDTHH:mm:ss[Z]',
    generalErrorKey: '__general__',
    apiPrefixV1: 'api/v1',
    projection: {toProjection: 'EPSG:3857', coordinates: [19, 52] as [number, number]},
    maxZoom: 12,
    timezone: Intl.DateTimeFormat().resolvedOptions().timeZone
}

# every other environment file eg. environment.hmr.private.ts

import sharedEnvironment from './base';
import commonEnvironment from './environment.common';

export const environment = {
version: 'undefined-dev-version',
   ...commonEnvironment
   ...sharedEnvironment,
   production: false,
   hmr: true,
   inviteOnly: true,
   uiDevelopment: true
}

```

This way we can change these parameters in one place, and not break other builds by accident (from what I understand those constants should be the same between builds)